app/models/git_tag.rb

```diff
@@ -0,0 +1,2 @@
+class GitTag < GitReference
```

Comment:
You're using subclassing, but not adding a type column to the table for STI...was that intentional?